src/routes.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (type = 'default') => {
+  switch (type) {
+    case 'default':
```

Comment:
Why does this need to be a separate file? These should be in-lined unless there's good reason otherwise

Reply:
Sure